src/common/Pagination/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Button from "./Button";
+import Icon from "./Icon";
+import PageInfo from './PageInfo';
+import { Footer } from './styled';
+
+const Pagination = () => {
+  return (
+    <Footer>
+      <Button
+        previous
+        buttonText="First"
+        disabled={true}
+        body={
+          <>
+            <Icon
+              mobile
+              previous
+              disabled={true}
+              alt="back to first page symbol"
+            />
+            <Icon
+              previous
+              disabled={true}
+              alt="back to previous page symbol"
+            />
+          </>}
+      />
+      <Button
+        previous
+        buttonText="Previous"
+        disabled={true}
+        body={
+          <>
+            <Icon
+              previous
+              disabled={true}
+              alt="back to first page symbol"
+            />
+          </>}
+      />
+
+      <PageInfo />
+      
+      <Button
+        buttonText="Next"
+        disabled={false}
+        body={
+          <>
+            <Icon
+              disabled={false}
+              alt="move to next page symbol"
+            />
+          </>}
+      />
+      <Button
+        buttonText="Last"
+        disabled={false}
+        body={
+          <>
+            <Icon
+              disabled={false}
+              alt="move to last page symbol"
+            />
+            <Icon
+              mobile
+              disabled={false}
+              alt="move to last page symbol"
+            />
+          </>}
+      />
+    </Footer>
+  )
+}
+
+export default Pagination;
```

Comment:
Trochę mnie kusi, żeby te sparametryzowane ikonki wynieść do oddzielnych komponentów, żeby ten kod był tutaj czystszy i żeby do body przekazywać gotowy komponent. Ale ta parametryzacja będzie zależeć od propsów z api, więc pewnie nie ma to sensu 🙈

Reply:
albooo... stworzyć całe "PreviousButton", "NextButton" itd i tam ustawiać propsy dla Buttona

Reply:
Ikonki są osobnym komponentem funkcyjnym. Nie bardzo widzę już sposób co by więcej z tym można zrobić :)

Reply:
ale może lepiej to na razie zostawić i ewentualnie zrefaktoryzować przy dodawaniu logiki

Reply:
myślałem zrobić osobny Button dla przed i po ale zdecydowałem się na jeden z odpowiednim parametrem i tak wydaje mi się, że jest lepiej

Reply:
Jeden uniwersalny jest super ale ja bym go jeszcze dodatkowo "rozszerzyła" i zrobiła wstecz i dalej, które miałyby go w sobie. Pewnie trochę przekombinowane, ale ja lubię jak te ostateczne komponenty nie mają w sobie za dużo i od razu na pierwszy rzut oka widać co tam jest, nie wczytując się w propsy. 
Ale tak jak teraz jest też jest spoko, więc może jak najbardziej zostać 👍